docs/api/XCastPlugin.md

```diff
@@ -358,7 +359,7 @@ The following table provides a client error mapping example:
 <a name="registerApplications"></a>
 ## *registerApplications*
 
-Registers an application. This allows to whitelist the apps which support dial service. To dynamically update the app list, same API should be called with the updated list.
+This API registers applications passed with XCast service. This allows to whitelist the apps which support dial service. Calling this api again, with a new set of APP names, will append those APP names to existing whitelist. Passing an existing whitelisted app, with a modified parameter value, will update the curresponding field in the whitelist.
```

Comment:
This API allows to application to whitelist the apps which support dial service with xcast service. Application can register single or multiple apps by passing application list.